101-recovery-services-backup-file-share/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "subscriptionId": {
+      "type": "string",
+      "defaultValue": "[subscription().subscriptionId]",
+      "metadata": {
+        "description": "The Subscription ID of the Subscription in which the Resource Group is present."
+      }
+    },
+    "resourceGroupName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "Name of the Resource Group in which the existing Storage Account is present."
+      }
+    },
+    "existingStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the existing Storage Account in which the existing File Share to be protected is present."
+      }
+    },
+    "existingFileShareName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the existing File Share to be protected."
+      }
+    },
+    "isNewVault": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Set to true if a new Recovery Services Vault is to be created; set to false otherwise."
+      }
+    },
+    "isNewPolicy": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Set to true if a new Backup Policy is to be created for the Recovery Services Vault; set to false otherwise."
+      }
+    },
+    "registerStorageAccount": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "Set to true if the existing Storage Account has to be registered to the Recovery Services Vault; set to false otherwise."
+      }
+    },
+    "vaultName": {
+      "type": "string",
+      "defaultValue": "RSVault1",
```

Comment:
does this need to be globally unique?  if so, use a globally unique defaultValue

Reply:
No, the vault name need not be globally unique.